core/series/supported.go

```diff
@@ -292,104 +286,120 @@ var ubuntuSeries = map[SeriesName]SeriesVersion{
 	Eoan: {
 		WorkloadType: ControllerWorkloadType,
 		Version:      "19.10",
-		Supported:    true,
 	},
 	Focal: {
 		WorkloadType: ControllerWorkloadType,
 		Version:      "20.04",
 		LTS:          true,
 		Supported:    false,
 	},
+	Groovy: {
+		WorkloadType: ControllerWorkloadType,
+		Version:      "20.10",
+		Supported:    true,
```

Comment:
This is wrong, this should be false.

If we don't support focal by default, as we had to make changes to juju to support focal correctly then we shouldn't support groovy either.

Reply:
The focal support issue was more so for bootstrap behaviour - what LTS would be chosen by default.
Nonetheless, I've made groovy = false